Submissions/yapalfredo.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Alfredo Yap",
+  "img": "https://avatars.githubusercontent.com/u/36459847?v=4",
+  "email": "al@alfredoyap.com",
+  "links": {
+    "website": "alfredoyap.com",
```

Comment:
Please provide the full URL

Reply:
oh.. I'm sorry about that.. on it. 

Thank you